arrows/core/video_input_image_list.cxx

```diff
@@ -354,169 +360,164 @@ ::frame_timestamp() const
   return ts;
 }
 
-
-// ------------------------------------------------------------------
-kwiver::vital::image_container_sptr
+// ----------------------------------------------------------------------------
+kv::image_container_sptr
 video_input_image_list
 ::frame_image()
 {
   if ( !d->m_image && this->good() )
   {
-    LOG_DEBUG( logger(), "reading image from file \"" << *d->m_current_file << "\"" );
+    LOG_DEBUG( logger(),
+               "reading image from file \"" << *d->m_current_file << "\"" );
 
-    // read image file
+    // Read image file
     //
-    // This call returns a *new* image container. This is good since
-    // we are going to pass it downstream using the sptr.
+    // This call returns a *new* image container; this is good since
+    // we are going to pass it downstream using the sptr
     d->m_image = d->m_image_reader->load( *d->m_current_file );
   }
   return d->m_image;
 }
 
-
-// ------------------------------------------------------------------
-kwiver::vital::metadata_vector
+// ----------------------------------------------------------------------------
+kv::metadata_vector
 video_input_image_list
 ::frame_metadata()
 {
-  if ( ! this->good() )
+  if ( !this->good() )
   {
-    return vital::metadata_vector();
+    return {};
   }
-  auto md = d->frame_metadata( *d->m_current_file, d->m_image );
-  vital::metadata_vector mdv(1, md);
-  return mdv;
-}
 
+  return { 1, d->frame_metadata( *d->m_current_file, d->m_image ) };
+}
 
-// ------------------------------------------------------------------
-kwiver::vital::metadata_map_sptr
+// ----------------------------------------------------------------------------
+kv::metadata_map_sptr
 video_input_image_list
 ::metadata_map()
 {
   if ( !d->m_have_metadata_map )
   {
-    kwiver::vital::timestamp::frame_t fn = 0;
-    for (const auto& f: d->m_files)
+    auto fn = kv::timestamp::frame_t{ 0 };
```

Comment:
I don't see the value of using `auto` in this case.

Reply:
Mostly [AAA](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/).